int32 x1
int32 y1
int32 x2
int32 y2
float32 confidence
string class_name
int32 class_id
int32 x
int32 y
int32 width
int32 height

